src/main/java/uk/nhs/hee/tis/trainee/forms/service/LtftService.java

```diff
@@ -478,4 +492,19 @@ private Query buildAdminFilteredQuery(Map<String, String> filterParams, Pageable
         });
     return query;
   }
+
+  /**
+   * Publish LTFT status update notification.
+   *
+   * @param form The updated LTFT form
+   */
+  private void publishStatusUpdateNotification(LtftForm form) {
+    log.info("Published status update notification for LTFT form {}", form.getId());
+    String groupId = form.getId() == null ? UUID.randomUUID().toString() : form.getId().toString();
```

Comment:
A form _should_ always have an id by this point, but to (lazily) avoid having to tweak all the tests that don't bother setting one, I've included a backstop. Always a good idea, a backstop.